### ISSUE TITLE
You're technically breaking §4 of the js.org ToS

### ISSUE THREAD

**Issue by NbOpposite** [discordjs/website#21]
https://js.org/terms.html

**§ 4 Rights and Duties of the user**
>[..]
>The user is obliged to publish legals on his website which contains at least the full name, mailing address and an e-mail adress.

I don't think these legalities are published anywhere on the main discord.js.org site.

**Comment by Gawdl3y** [comment]
That was added in September, long after we began using js.org... I suspect the majority those terms are there more for their protection than anything else. Regardless, I'll contact them and see what they suggest.

**Comment by amishshah** [comment]
I'd be happy to put my name and e-mail address but not mailing address FWIW

**Comment by NbOpposite** [comment]
Yeah, I find the mailing address part a bit over the top, although the rest we should be able to comply with.

**Comment by Gawdl3y** [comment]
@indus Mind commenting on this?

**Comment by indus** [comment]
This obligation comes from a german law called "Telemedien Gesetz" that includes regulations about websites like the one you are discussing about. It was in place long before GDPR and is still relevant on a national level. I'm not checking if everybody fullfills this term because its just not possible (it would be possible for requesters to fake the information, or remove it after I have added the subdomain).
The terms of service should just make sure I don't get sued for the missbehaviour of some subdomain owner (and to make sure nobody ruins the reputation - piracy, phishing, etc.). Nevertheless JS.ORG is subject to german law - weather this is true or not for every website with a JS.ORG subdomain is debatable (propably not).

In short: Do what you would do with your personal website - don't use JS.ORGs subdomain for shady business - stick to the laws (that apply to you)
If your website makes trouble to JS.ORG because of legal issues, I won't hire a lawyer just be prepared to loose the subdomain.

I hope that helps.

**Comment by indus** [comment]
> That was added in September, long after we began using js.org... I suspect the majority those terms are there more for their protection than anything else. Regardless, I'll contact them and see what they suggest.
That is not correct. The Github history may be not clear on that but this paragraph is there since the beginning in 2015.

**Comment by BigBrainAFK** [comment]
AFAIK the TMG only affects websites with their headquarters inside of Germany. In EU states you have to use the country of origin rules which means that you just need to follow the laws of the country that the headquarters reside in. For non-EU states theres no ruling in the TMG itself so atm they don't need to follow it either. You can however provide a contact mail and name just in case. A complete mailing address could then be mentioned to be handed out if requested so its not just publicly visible but that the contact person has control over that information.

**Comment by kav2k** [comment]
Needed to empty my cache for it to work. Apologies. But it's not obvious.
